examples/proof-of-concept/certificate-management/things/cmd/cert-management/main.go

```diff
@@ -202,41 +152,20 @@ func requestCertificate(device thing.Thing) {
 func main() {
 	//thing.SetDebugLogger(log.New(os.Stdout, "", log.Ldate|log.Ltime|log.Lmicroseconds|log.Llongfile))
 	deviceID := "Device-8456232771"
-	amURL, _ := url.Parse(AmUrl)
-	var cnfKey jose.JSONWebKey
+	amURL, _ := url.Parse(os.Getenv("AM_URL"))
 	store := secrets.Store{}
-	signer, err := store.Signer(deviceID)
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
-	}
-	cnfKey = jose.JSONWebKey{Key: signer.Public(), Algorithm: string(jose.ES256), Use: "sig"}
-	kid, err := cnfKey.Thumbprint(crypto.SHA256)
+	thingKey, err := store.Signer(deviceID)
 	if err != nil {
 		fmt.Println(err)
 		os.Exit(1)
 	}
-	cnfKey.KeyID = base64.URLEncoding.EncodeToString(kid)
-	keySet := jose.JSONWebKeySet{
-		Keys: []jose.JSONWebKey{
-			{
-				Key:   signer.Public(),
-				KeyID: cnfKey.KeyID,
-				Use:   "sig",
-			},
-		},
-	}
-	b, _ := json.Marshal(keySet)
+	thingKid, _ := thing.JWKThumbprint(thingKey)
 
```

Comment:
With this change we've lost the ability to demo the rotate functionality. To reintroduce it, the following lines should be:
```
        fmt.Println("\nPress Enter to register and authenticate...")
	fmt.Scanln()
	device := register(deviceID, amURL, thingKid, thingKey)
	fmt.Println("\nPress Enter to request the certificate...")
	fmt.Scanln()
	requestCertificate(device)
        
        fmt.Println("\nPress Enter to re-authenticate and request the certificate...")
	fmt.Scanln()
	device = authenticate(deviceID, amURL, thingKid, thingKey)
	requestCertificate(device)
```

You will have to reintroduce the authenticate function and update the parameters to be the same as the register function.